packages/block-editor/src/components/block-content-overlay/content.scss

```diff
@@ -37,7 +37,7 @@
 		pointer-events: none;
 	}
 
-	.block-editor-iframe__body.is-zoomed-out &::before {
+	.edit-site-visual-editor__editor-canvas.is-zoomed-out &::before {
```

Comment:
This style is now applied to the iframe right? Then it should move out of content.css because it's no longer a content style

Reply:
I think that style is wrong, it should be `.edit-site-visual-editor__editor-canvas.is-zoomed-out body &::before {` instead. Or probably just `iframe[name="editor-canvas"].is-zoomed-out body &::before {`

Reply:
I need to double check